docs/org.eclipse.n4js.design/build.properties

```diff
@@ -0,0 +1 @@
+bin.includes = META-INF/
```

Comment:
File `docs/org.eclipse.n4js.design/.project` is missing, so new project cannot be imported into Eclipse workspace. Same for settings folder (best copy that from plugin containing the spec).

Reply:
added along with .settings and .classpath from spec